### PR TITLE
59178 allow google backend login

### PR DESCRIPTION
**Pull request by rnuernberg**
https://github.com/chameleon-system/chameleon-system/pull/807 must be merged first

## Summary by CodeRabbit

- **New Features**
  - Integrated OAuth2 client support for improved authentication options.

- **Refactor**
  - Streamlined user profile editing process for enhanced performance and reliability.

- **Chores**
  - Updated application kernel with the latest bundle configurations.

- **Bug Fixes**
  - Removed deprecated bundles to ensure system stability and security.

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -11,14 +11,48 @@
...
+        },
+        {
+            "type": "path",
+            "url": "packages/chameleon-system/sanitycheck"
+        },
+        {
+            "type": "path",
+            "url": "packages/chameleon-system/sanitycheck-bundle"
+        }
+    ],
```
Comment (bestform) [inline composer.json:40]: Those should not be in here i guess

File: composer.json [hunk 2]
```diff
@@ -11,14 +11,48 @@
...
     "require": {
-        "chameleon-system/chameleon-shop": "~7.1.0",
-        "chameleon-system/chameleon-shop-theme-bundle": "~7.1.0",
+        "chameleon-system/chameleon-base": "7.2.x-dev as 7.1.99",
+        "chameleon-system/chameleon-shop": "7.2.x-dev as 7.1.99",
+        "chameleon-system/chameleon-shop-theme-bundle": "7.2.x-dev as 7.1.99",
```
Comment (bestform) [inline composer.json:44]: Those should be master, not 7.2.x

File: composer.json [hunk 3]
```diff
@@ -11,14 +11,48 @@
...
         "incenteev/composer-parameter-handler": "~2.0",
-        "sensio/framework-extra-bundle": "*"
+        "symfony/security-bundle": "^5.4"
```
Comment (bestform) [inline composer.json:46]: This comes with chameleon core, doesn't it?

**Review by bestform — CHANGES_REQUESTED** [review]

File: composer.json (hunk 1 above)
Comment (rnuernberg) [reply composer.json:40]: ok

File: composer.json (hunk 2 above)
Comment (rnuernberg) [reply composer.json:44]: changed

File: composer.json (hunk 3 above)
Comment (rnuernberg) [reply composer.json:46]: true

**Pull request merged**